Networking/Networking/Model/OrderItemAttribute.swift

```diff
@@ -0,0 +1,33 @@
+public struct OrderItemAttribute: Decodable, Hashable, Equatable {
+    public let metaID: Int64
+    public let name: String
+    public let value: String
```

Comment:
nit: Is it maybe better to rename it as `displayValue` since there is another field called `value` that maybe in the future we want to parse?

Reply:
I'd prefer adding just the fields we need now and add new ones if we need them in the future 🙂 also, there's only `name` and `value` and in the storage layer so it'd be nice keeping them consistent